modules/users/server/config/strategies/twitter.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(config) {
 			var providerUserProfile = {
 				displayName: profile.displayName,
 				username: profile.username,
-				profileImageURL: (profile.photos && profile.photos.length) ? profile.photos[0].value : undefined,
+				profileImageURL: (profile.photos && profile.photos.length) ? profile.photos[0].value.replace('normal', 'bigger') : undefined,
```

Comment:
Right, but is that in the array? If it is, then we don't need to check the length and existence.